cloudify_cli/commands/plugins.py

```diff
@@ -560,32 +561,48 @@ def update(blueprint_id,
             '--to-minor are mutually exclusive.  If you want to upgrade '
             'only the specific plugins, use --plugin-name parameter instead.')
 
-    utils.explicit_tenant_name_message(tenant_name, logger)
     if blueprint_id:
-        _update_a_blueprint(blueprint_id, all_tenants, plugin_names,
-                            to_latest, all_to_latest, to_minor, all_to_minor,
-                            include_logs, json_output, logger,
-                            client, force, auto_correct_types,
-                            reevaluate_active_statuses)
+        _update_a_blueprint(blueprint_id,
+                            plugin_names,
+                            to_latest,
+                            all_to_latest,
+                            to_minor,
+                            all_to_minor,
+                            include_logs,
+                            json_output,
+                            logger,
+                            force,
+                            auto_correct_types,
+                            reevaluate_active_statuses,
+                            client,
+                            tenant_name)
```

Comment:
```suggestion
        _update_a_blueprint(
            blueprint_id,
            plugin_names,
            to_latest,
            all_to_latest,
            to_minor,
            all_to_minor,
            include_logs,
            json_output,
            logger,
            force,
            auto_correct_types,
            reevaluate_active_statuses,
            client,
            tenant_name,
        )
```
I suggest this style: you say you want the arguments to be one-line-each, so why not go even one step further, and have indeed _all_ the args one-line-each, even the last one: with the trailing comma and the `)` on the separate line